CHANGELOG.md

```diff
@@ -1,3 +1,42 @@
+# 1.7.1 (November 2024)
+
+_Changes since 1.7.0_
+
+## Fixes
+
+### Multiple Platforms
+
+- [Fixed `Modifier.clickable` binary compatibility with 1.6 on non-JVM targets](https://github.com/JetBrains/compose-multiplatform-core/pull/1647)
+- [Fixed `Modifier.toggleable` and `Modifier.selectable` binary compatibility with 1.6 on non-JVM targets](https://github.com/JetBrains/compose-multiplatform-core/pull/1649)
+- [Fix Skia paragraph caching performance degradation](https://github.com/JetBrains/compose-multiplatform-core/pull/1676)
+
+### iOS
+
+- [Fling animation works correctly for fast scrolling gestures](https://github.com/JetBrains/compose-multiplatform-core/pull/1616)
+- [Fix HorizontalPager snapping on iOS](https://github.com/JetBrains/compose-multiplatform-core/pull/1661)
+- [Fixed double recomposition on the first screen](https://github.com/JetBrains/compose-multiplatform-core/pull/1668)
+- [Fix Accessibility Items availability inside dialogs](https://github.com/JetBrains/compose-multiplatform-core/pull/1678)
+- [Update DatePickerDialog to the latest AOSP version](https://github.com/JetBrains/compose-multiplatform-core/pull/1666)
```

Comment:
The description doesn't match with PR's one. "Update to AOSP" is incorrect for change log "Fix issue where `DateRangePicker` doesn't show confirmation button" is. Also, it's not iOS specific.

Reply:
Fixed